src/drawing/backend_impl/bitmap.rs

```diff
@@ -231,6 +231,11 @@ pub trait PixelFormat: Sized {
                         *buf.get_unchecked_mut(base + idx) = Self::byte_at(r, g, b, 0, idx);
                     }
                 } else {
+                    let alpha = alpha.min(1.0).max(0.0);
```

Comment:
Just wondering isn't this already checked to be less than 1.0 already? There might be a overflow, but it seems what we should do is changing the if condition from `1.0` to `255.0 / 256.0`?